install.sh

```diff
@@ -26,40 +48,40 @@ function install-theme() {
 
 	for dir in "${steamdirs[@]}"; do
 		if [ -d "$dir" ]; then
-			read -p "Would you like to remove any previous installation of catppuccin? [y/n] " -n 1 -r
+			read -p "Would you like to remove any previous installation of Catppuccin? [y/n] " -n 1 -r
 			echo
 			if [[ $REPLY =~ ^[Yy]$ ]]; then
-				rm -fr "$dir/steam/skins/catppuccin-"*
+				rm -fr "$dir/steam/skins/Catppuccin "*
 			fi
 			cd "$(mktemp -d)"
-			mkdir -p "$dir/steam/skins/catppuccin-$opt"
-			install_path="$dir/steam/skins/catppuccin-$opt"
+			mkdir -p "$dir/steam/skins/Catppuccin $opt"
```

Comment:
Spaces in the skin name is asking for trouble, it would be better for it to be a dash or underscore